### PR TITLE
eRegs progressive web app

### PR DESCRIPTION
**Pull request by contolini**
This is the same as #4049 but with the URLs updated to accommodate the new location at `cf.gov/regulations`. 

## Additions

- Custom webpack config file for eRegs
- Service worker generating script in the webpack file (the generated service worker is ignored by git)
- Unofficial-definitely-not-approved-but-still-cool eRegs icons designed by the amazing @niqjohnson 
- A [web app manifest](https://developers.google.com/web/fundamentals/web-app-manifest/) that tells browsers about the website

## Changes

- Use [routablepageurl](http://docs.wagtail.io/en/v2.0.1/reference/contrib/routablepage.html#the-routablepageurl-template-tag) instead of hardcoding prev/next URLs.

## Testing

1. `./frontend.sh`
1. Load a new Regulations 3k reg section page.
1. Open the Application tab in chrome dev tools and see if the manifest and service worker are present.

## Screenshots

screen shot 2018-04-24 at 1 59 23 am

screen shot 2018-04-24 at 1 59 38 am

As you click through a reg's sections, in the Network tab you should see the next page being prefetched and cached by the service worker (gear icon = service worker).

screen shot 2018-04-30 at 9 55 37 pm

### CODE REVIEW

File: cfgov/regulations3k/tests/test_models.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def test_routable_page_get_context(self):
 
     def test_get_reg_nav_items(self):
         request = HttpRequest()
-        request.url = '/eregulations3k/1002/1002-4/'
+        request.url = '/regulations/1002/1002-4/'
```
Comment (higs4281) [inline cfgov/regulations3k/tests/test_models.py]: probably doesn't matter for the test, but we're using the form `/1002/4/` now.

File: cfgov/unprocessed/apps/regulations3k/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cfpb-regulations-3000",
+  "name": "eRegulations",
   "description": "The consumerfinance.gov/eregulations website.",
   "homepage": "https://www.consumerfinance.gov/eregulations/",
```
Comment (higs4281) [inline cfgov/unprocessed/apps/regulations3k/package.json]: Do we want to go with `/regulations/` everywhere for now?

Comment (contolini) [reply cfgov/unprocessed/apps/regulations3k/package.json]: Yesh

File: cfgov/regulations3k/tests/test_models.py (hunk 1 above)
Comment (contolini) [reply cfgov/regulations3k/tests/test_models.py]: Good call

File: cfgov/unprocessed/apps/regulations3k/package.json (hunk 2 above)
Comment (higs4281) [reply cfgov/unprocessed/apps/regulations3k/package.json]: homepage too?

**Pull request merged**